Podfile

```diff
@@ -13,8 +13,8 @@ use_frameworks!
 # - `{ {kit spec hash} => {sdk spec hash}` to depend on specific pod options (:git => …, :podspec => …) for each repo. Used by Fastfile during CI
 #
 # Warning: our internal tooling depends on the name of this variable name, so be sure not to change it
-$matrixKitVersion = '= 0.16.9'
-# $matrixKitVersion = :local
+# $matrixKitVersion = '= 0.16.9'
+$matrixKitVersion = :local
```

Comment:
Upsie, committed these by mistake. I'll fix it before I merge, I need to disable the build setting too.